src/System.Runtime.Serialization.Formatters/src/System/Runtime/Serialization/ObjectManager.cs

```diff
@@ -1624,4 +1624,17 @@ internal TypeLoadExceptionHolder(string typeName)
 
         internal string TypeName { get; }
     }
+
+    // TODO: Temporary workaround.  Remove this once SerializationInfo.UpdateValue is exposed
```

Comment:
These were added by @stephentoub and I'm not sure if he logged issues for it. I'll add some myself.